src/masternodes/rpc_proposals.cpp

```diff
@@ -611,7 +612,7 @@ UniValue votegovbatch(const JSONRPCRequest &request) {
     RPCTypeCheck(request.params, {UniValue::VARR}, false);
 
     const auto &keys = request.params[0].get_array();
-    auto sleepTime = gArgs.GetBoolArg("-sleep-time", SLEEP_TIME_MILLIS);
+    auto sleepTime = request.params[1].isNull() ? SLEEP_TIME_MILLIS : request.params[1].get_int();
```

Comment:
I think you need to check if the param is set first, otherwise the node may crash.

Reply:
.isNull() is the check.